src/app/services/error-handler.service.ts

```diff
@@ -0,0 +1,249 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorHandler, Injectable, Injector } from '@angular/core';
+import { AbstractControl, UntypedFormGroup } from '@angular/forms';
+import { TranslateService } from '@ngx-translate/core';
+import * as Sentry from '@sentry/angular';
+import { Observable } from 'rxjs';
+import { ResponseErrorType } from 'app/enums/response-error-type.enum';
+import { sentryCustomExceptionExtraction } from 'app/helpers/error-parser.helper';
+import { ErrorReport } from 'app/interfaces/error-report.interface';
+import { Job } from 'app/interfaces/job.interface';
+import { WebsocketError } from 'app/interfaces/websocket-error.interface';
+import { DialogService } from 'app/services';
+
+@Injectable()
+export class ErrorHandlerService implements ErrorHandler {
```

Comment:
This class mixes a lot of responsibilities:
* Default Angular error handler (Sentry reporting).
* Parsing errors.
* Showing form errors.
* Reporting generic errors.

It feels like these should be 4 separate things, not one.